packages/grafana-data/src/dataframe/ArrowDataFrame.ts

```diff
@@ -173,6 +177,15 @@ export function grafanaDataFrameToArrowTable(data: DataFrame, keepOriginalNames?
   return table;
 }
 
+export function arrowTableToBase64String(table: Table): string {
```

Comment:
@aocenas I think this undocumented export is one of the reasons you have the API docs failure in drone